models/contact.py

```diff
@@ -0,0 +1,119 @@
+import time
+
+from core.citext import CIText
+from core.db import db
+
+
+class Challenge(db.Model):
+    ## States
```

Comment:
enums don't exist in python2

(also, the values matter since a positive number represents the amount of time left)